### PR TITLE
Update and cygwin fix2

### PR DESCRIPTION
**Pull request by saulery**
Hello,

I have reworked my PR. 

I had to switch core.autocrlf and core.filemode to false.

Regards,

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -716,9 +825,9 @@ crt: tools
 
 	rm *.tmp
 
-CRYPTLIB := cl343_beta
+CRYPTLIB := cl344
```
Comment (dkl) [inline makefile:828]: This seems to change the binding, so we should update that too.

**Comment by dkl** [comment]
Ok thanks, looks pretty good!

**Comment by dkl** [comment]
Hmm, could you please rebase/cherry-pick your branch on top of the raw branch, instead of master? The idea was that raw holds the scripts & generated bindings, while master holds that plus additional manual modifications.

**Comment by saulery** [comment]
I'm not a git guru. I don't know how to rebase all my branch on another branch. Cherry picking on another branch seem easier. I'm in travel and can do it monday only.

For cl344 I keep thing as is, or I revert and put a not in a TODO file ?

**Comment by dkl** [comment]
If you don't mind I will do the cherry-pick and merge the changes that way.
As for cryptlib, ideally the bi files are updated in parallel with the script (until there is a better mechanism). It looks easy enough so I will give it a try. Otherwise it would be best if you could send a pull request with the updated bi files later.

**Pull request closed without merging**